tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+| Public Domain         | This section is to explicitly link to Federal policies and guidelines that required or recommended for Federal projects to comply with, such as Accessibility (508) Interoperability, Anti-deficiency, Security, Licensing, and other policies that can vary between agencies and domains.          |              |
+
+- [ ] **repolinter.json**
+_Lints repository for missing files and sections above_
+
+**Results**
+*Insert review here*
+
+
+----
+
+
+### Review Repository Details
+
+The GitHub repository homepage features a concise description of the project, a list of relevant topic tags, and general information about the repository to provide a comprehensive overview for users and contributors.
+
+_About Section_:
+
+- [ ] *Description*
+  1-2 sentences describing the project
+
+- [ ] *Website*
+Link to project's website
+
+- [ ] *Topics*
+Tags for project discoverability. Helpful topics to classify a repository include the repository's intended purpose, subject area, community, or language.
+
+_Include in Home Page_:
+
+- [ ] Releases
+- [ ] Packages
+- [ ] Deployments
+
+
+
+**Results**
+
+*Insert review here*
+
+
+
+----
+
+
+
+### Additional Notes & Questions
+
+"*Insert any notes or questions here*"
+
+
+
+
+
+### Sign off on risk acceptance of open-sourcing the software product
+
+After reviewing the materials prepared by the team that is working to open source the product, the business owner signs off on a risk acceptance for open-sourcing the software product.
+
+
+Requesting sign off from key people on this request.
+
+
+| **Reviewer Organization**      | **Reviewer Name**                                  | Reviewer's Recommandation**                                                     |
+|--------------------------------|----------------------------------------------------|---------------------------------------------------------------------------------|
+| Code Reviewer's Recommendation | CODE REVIEWER 1  CODE REVIEWER 2   CODE REVIEWER 3 | [Approved/Needs Approval]  [Approved/Needs Approval]  [Approved/Needs Approval] |
+| ISSO                           | ISSO REVIEWER                                      | [Approved/Needs Approval]                                                       |
+| ISG Technical Approval         | ISG REVIEWER                                       | [Approved/Needs Approval]                                                       |
+| Business Owner(s)              | BUSINESS OWNER 1  BUSINESS OWNER 2                 | [Approved/Needs Approval]  [Approved/Needs Approval]                            |
+
+
+
+---
+
+
+
+### Flipping the Switch: Making the Repository Public
+
+Once the repository has passed outbound review, we are ready 
```

Comment:
This is missing some text. Should be: 

Once the repository has passed outbound review, we are ready to “flip the switch” and officially make it public. Once made public, there are a couple of actions that need to be taken:

Reply:
I don't know how I missed an entire chunk!